octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -707,7 +709,17 @@ async def order(self, order: cryptofeed_types.OrderInfo, receipt_timestamp: floa
         :param order the order object defined in cryptofeed.types.?
         :param receipt_timestamp: received timestamp
         """
-        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, orders=[self.exchange.parse_order(order)])
+        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, orders=[
+            personal_data.create_order_instance(trader=self.exchange_manager.trader,
+                                                order_type=self._parse_order_type(order.type),
+                                                status=self._parse_order_status(order.status),
+                                                order_id=order.id,
+                                                symbol=order.symbol,
+                                                current_price=order.price,
+                                                quantity=order.amount,
+                                                quantity_filled=order.amount - order.remaining,
```

Comment:
is it not supposed to be a dict and it's in the ordre channel that we create the actual order ?